src/main/java/nextstep/fp/StreamStudy.java

```diff
@@ -15,19 +16,22 @@ public static long countWords() throws IOException {
                 .get("src/main/resources/fp/war-and-peace.txt")), StandardCharsets.UTF_8);
         List<String> words = Arrays.asList(contents.split("[\\P{L}]+"));
 
-        long count = 0;
-        for (String w : words) {
-            if (w.length() > 12) count++;
-        }
-        return count;
+        return words.stream().filter(w -> w.length() > 12).count();
```

Comment:
```suggestion
        return words.stream()
                .filter(w -> w.length() > 12)
                .count();
```
stream 관련 메소드는 한 줄에 한 개만 사용해주세요!